notebook/worker-template.yml

```diff
@@ -2,19 +2,12 @@ metadata:
 spec:
   restartPolicy: Never
   containers:
-  - args:
-      - dask-worker
-      - --nthreads
-      - '1'
-      - --no-bokeh
-      - --memory-limit
-      - 11.5GB
-      - --death-timeout
-      - '60'
+  - args: [dask-worker, $(DASK_SCHEDULER_ADDRESS), --nthreads, '1', --no-bokeh,
+          --memory-limit, 11.5GB, --death-timeout, '60']
     env:
       - name: GCSFUSE_BUCKET
         value: rhg-data
-    image: rhodium/worker:d15fbe5fa4c4f78740cd3ab566007adf2f801947
+    image: rhodium/worker:dev-update-packages
```

Comment:
for trivia - this actually gets overwritten with the current commit hash or tag by our build process and never sees the light of day. we should probably replace this with TRAVIS_REPLACE_ME for clarity.

Reply:
oh right! I'll just do that now.